sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -1035,7 +1035,7 @@ static size_t _iphc_ipv6_encode(gnrc_pktsnip_t *pkt,
 
         /* copy remaining byteos of flow label */
         iphc_hdr[inline_pos++] = (uint8_t)((ipv6_hdr_get_fl(ipv6_hdr) & 0x0000ff00) >> 8);
-        iphc_hdr[inline_pos++] = (uint8_t)((ipv6_hdr_get_fl(ipv6_hdr) & 0x000000ff) >> 8);
+        iphc_hdr[inline_pos++] = (uint8_t)((ipv6_hdr_get_fl(ipv6_hdr) & 0x000000ff) >> 0);
```

Comment:
Not really fighting for this. ;-)

```suggestion
        iphc_hdr[inline_pos++] = (uint8_t)(ipv6_hdr_get_fl(ipv6_hdr) & 0x000000ff);
```

Reply:
I thought I'd keep the symmetry, but looks like this very symmetry masked a bug.